app/scripts/services/ResourceFactoryProvider.js

```diff
@@ -26,7 +26,8 @@
                         getAllRoles: {method: 'GET', params: {}, isArray: true},
                         deleteRoles: {method: 'DELETE'},
                         disableRoles: {method: 'POST'},
-                        enableRoles: {method: 'POST'}
+                        enableRoles: {method: 'POST'},
+                        put: {method: 'PUT'}
```

Comment:
Rename this to `update` and then change the code accordingly. Let's see what happens with travis